### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Mrgaton** [cefsharp/CefSharp#4156] [General]
i have this error

[0709/204300.353:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204300.353:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 1 time(s)
[0709/204301.036:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0709/204301.994:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204301.995:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 2 time(s)
[0709/204302.606:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0709/204303.783:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204303.783:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 3 time(s)
[0709/204304.517:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0709/204305.460:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204305.460:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 4 time(s)
[0709/204306.160:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0709/204306.963:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204306.963:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 5 time(s)
[0709/204308.180:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0709/204308.313:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-532462766
[0709/204308.313:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 6 time(s)
[0709/204308.313:FATAL:gpu_data_manager_impl_private.cc(451)] GPU process isn't usable. Goodbye.

**Comment by amaitland** [comment]
Please provide more information see https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

Does the problem reproduce with the minimal example?

**Comment by Mrgaton** [comment]
The version of csharp is 
![image](https://user-images.githubusercontent.com/68958481/178592748-6663d000-cf36-437d-a925-543ed707a0b1.png)
i im using WinForms
im using net framework 4.8
the is no exception only the messages thath i showed

i tried to reinstall my grafics driver and didnt work

**Reply by amaitland** [reply]
In future no images of versions or errors. Just provide the answers as text. 

Also post all the answers in one, spread out like they are makes it harder to piece together.

**Comment by Mrgaton** [comment]
![image](https://user-images.githubusercontent.com/68958481/178599529-7e6b0f85-84ef-46c1-a739-c9f8b0264bf6.png)

**Comment by Mrgaton** [comment]
`
            string CefCachePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "CefSharp\\Cache");


            if (Directory.Exists(CefCachePath))
            {
                Directory.Delete(CefCachePath, true);
            }

            var settings = new CefSettings()
            {
                UserAgent = Compresor.DecompressString("882vyszJSdQ31TNQ0AjPzEvJLy9W8AtRMDTQM7BWAAqYmVgrVJiZaCo4FhTkpIanJnlnluibGpvrGZspaHh7hPj66CjkZGanKrinJmfnayo4ZxTl56bqW1roGeiZmJta6BkaGCkEJ6YlFmVCtQEA"),
                CachePath = CefCachePath,
            };
            MessageBox.Show("3re3r ");


            settings.LogSeverity = LogSeverity.Warning;
            settings.LogFile = Path.GetTempPath() + "browser.log";

            settings.CefCommandLineArgs.Add("enable-media-stream", "1");
            settings.CefCommandLineArgs.Add("enable-usermedia-screen-capturing", "1");
            settings.CefCommandLineArgs.Add("allow-http-screen-capture", "1");
            settings.CefCommandLineArgs.Add("enable-experimental-web-platform-features", "1");
            settings.CefCommandLineArgs.Add("use-fake-ui-for-media-stream", "1");
            settings.CefCommandLineArgs.Add("log-file", settings.LogFile);

            //settings.CefCommandLineArgs.Add("enable-gpu", "1");
            //settings.CefCommandLineArgs.Add("enable-webgl", "1");
            settings.CefCommandLineArgs.Add("enable-begin-frame-scheduling", "1");
            settings.CefCommandLineArgs.Add("--off-screen-frame-rate", "60");
            settings.CefCommandLineArgs.Add("off-screen-rendering-enabled", "1");


            settings.CefCommandLineArgs.Add("allow-running-insecure-content", "1");
            settings.CefCommandLineArgs.Add("enable-speech-input", "1");
            settings.CefCommandLineArgs.Add("enable-usermedia-screen-capture", "1");
            settings.CefCommandLineArgs.Add("debug-plugin-loading", "1");


            settings.CefCommandLineArgs.Add("allow-outdated-plugins", "1");
            settings.CefCommandLineArgs.Add("disable-web-security", "1");
            settings.CefCommandLineArgs.Add("enable-npapi", "1");
            settings.CefCommandLineArgs.Add("multi-threaded-message-loop", "1");


            settings.SetOffScreenRenderingBestPerformanceArgs();
            settings.DisableGpuAcceleration();

            Cef.Initialize(settings);


            if (Bot == false)
            {
                chromeBrowser = new ChromiumWebBrowser("https://discord.com");
                
                chromeBrowser.BrowserSettings = new CefSharp.BrowserSettings()
                {
                    LocalStorage = CefState.Enabled,
                    RemoteFonts = CefState.Enabled,
                    ImageLoading = CefState.Enabled,
                    Databases = CefState.Enabled,
                    JavascriptAccessClipboard = CefState.Enabled,
                    Javascript = CefState.Enabled,
                    WebGl = CefState.Disabled,
                    JavascriptCloseWindows = CefState.Enabled

                };
                MessageBox.Show("Iniciando ");

                chromeBrowser.Load("https://discord.com/app");

                this.Controls.Add(chromeBrowser);
                chromeBrowser.MenuHandler = new MyCustomMenuHandler();
                chromeBrowser.KeyboardHandler = new CEFKeyBoardHander();
                chromeBrowser.LifeSpanHandler = new MyCustomLifeSpanHandler();


                chromeBrowser.Name = ("Discord");
                chromeBrowser.Dock = DockStyle.Fill;
                chromeBrowser.ExecuteScriptAsyncWhenPageLoaded(MensajedBlockScript);

                await Task.Delay(1000);

                while (chromeBrowser.IsLoading)
                {
                    await Task.Delay(300);
                }
                await Task.Delay(300);

                chromeBrowser.ExecuteScriptAsync(Script + "\nlogin('" + Token.Replace("\n", "").Replace(" ", "") + "')");
            }

`

this is the code

**Comment by Mrgaton** [comment]
I don't know why but now it doesn't happen sorry for the inconvenience

**Comment by Mrgaton** [comment]
I have a question how can I make my browser have all or almost all the features that a normal browser has, I mean all the CefCommandLineArgs that I have to add to make it look like chrome because I use it in discord and there are still things that I don't they work in cefsharp

**Reply by amaitland** [reply]
Looks like you've hacked together all the command line args you could find, at least 7 will do nothing. Others shouldn't be used. The Setting you've enabled should already be enabled by default. 

Use LoadUrlAsync instead of load and await the result rather than looping. 

The alloy runtime which is used by default only implentations a subset of the chrome features. See #4123

**Comment by Mrgaton** [comment]
Load url async didnt work for me i onlyg et black screen

**Reply by amaitland** [reply]
Likely a mistake in your code.

**Comment by Mrgaton** [comment]
look for example in a normal web navigator like chrome the gifs looks like this
![image](https://user-images.githubusercontent.com/68958481/178795311-099d38e4-c025-4fd3-9a9f-4f9a55f3b705.png)


but in cefsharp look like this
![image](https://user-images.githubusercontent.com/68958481/178795372-bd4742e2-37b4-4bcc-834b-0b63eb071830.png)

**Reply by amaitland** [reply]
Entirely meaningless without specific details. 

Use the minimal example for testing. https://github.com/cefsharp/CefSharp.MinimalExample
